package.json

```diff
@@ -2,6 +2,7 @@
   "name": "react-formal",
   "version": "0.17.9",
   "description": "Classy HTML form management for React",
+  "jsnext:main": "src/index.js",
```

Comment:
I'm looking at tree shaking and general benefits of navigating code while developing.